<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundSignalDialog</class>
 <widget class="QDialog" name="SoundSignalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="apply_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_frequency_measurement_unit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>4700 Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period_measurement_unit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>3 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_measurement_unit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>50 %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_duration_measurement_unit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1 s</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="singleStep">
    <number>0</number>
   </property>
   <property name="value">
    <number>3000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="period_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Period</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequency_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>321</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>700</number>
   </property>
   <property name="maximum">
    <number>4700</number>
   </property>
   <property name="value">
    <number>4700</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="duration_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>321</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>321</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>235</green>
         <blue>226</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>235</green>
         <blue>226</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="alert_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ON/OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="alert_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alert</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>235</green>
         <blue>226</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>235</green>
         <blue>226</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="battaries_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ON/OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="battaries_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Batteries</string>
   </property>
  </widget>
  <widget class="QLabel" name="volume_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
